data/core/chitchat.md

```diff
@@ -30,22 +30,17 @@
 * ask_whatspossible
     - action_chitchat
 
-
 ## just newsletter + confirm
 * greet
     - action_greet_user
 * ask_weather OR ask_builder OR ask_howdoing OR ask_whoisit OR ask_whatisrasa OR ask_isbot OR ask_howold OR ask_languagesbot OR ask_restaurant OR ask_time OR ask_wherefrom OR ask_whoami OR handleinsult OR nicetomeeyou OR telljoke OR ask_whatismyname OR howwereyoubuilt
     - action_chitchat
 * signup_newsletter
-    - utter_great
-    - utter_ask_email
-* enter_data{"email": "maxmeier@firma.de"} OR enter_data{"number":"1"}
-    - action_store_email
-    - slot{"email": "maxmeier@firma.de"}
-    - action_subscribe_newsletter
-    - slot{"subscribed": true}
-    - utter_awesome
-    - utter_confirmationemail
+    - utter_can_do
+    - subscribe_newsletter_form
+    - form{"name": "subscribe_newsletter_form"}
+    - slot{"email": "ella@ella.ella"}
```

Comment:
If the slot is featurized, it has to be there right? I kept this slot featurized so that we could keep the flow from before where "sign up ella@ella.ella" works and doesn't re-ask the email.

Reply:
but maybe i'm misunderstanding the slot events and what they do

Reply:
no your understanding of slot events is correct. But we can unfeaturize this, because the form can automatically handle it when users provide an email. I think i commented on that below

Reply:
I tested out the unfeaturized shortly this morning and it seemed to work :)